google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/DefaultCloudSdkService.java

```diff
@@ -40,4 +83,206 @@ public String getCloudSdkHomePath() {
   public void setCloudSdkHomePath(String cloudSdkHomePath) {
     propertiesComponent.setValue(CLOUD_SDK_PROPERTY_KEY, cloudSdkHomePath);
   }
+
+  @Nullable
+  private Path getJavaToolsBasePath() {
```

Comment:
my reasoning was that I didn't want the IDE to explode if gcloud ever changes the way it packages these libs. But maybe having it explode would be a way for us to quickly be informed. What do you think?